sim_nav_goals:
  table:
    position:
      x: -2.9408016204833984
      y: -1.3139156103134155
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.9152988066802662
      w: 0.402775488938541
  chair:
    position:
      x: 2.521484613418579
      y: -2.043757200241089
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.7040760069197588
      w: 0.7101246203871034
  doorway_facing_green:
    position:
      x: 0.3962286114692688
      y: 0.09294455498456955
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.9999957096443621
      w: 0.0029292819715136826
  doorway_facing_blue:
    position:
      x: -0.6024703979492188
      y: 0.1385389119386673
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.004361087136426318
      w: 0.9999904904142781
  large_sofa:
    position:
      x: -4.568149566650391
      y: 0.24872761964797974
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.9999849932886496
      w: 0.005478430203923843
  small_sofa:
    position:
      x: -1.1403510570526123
      y: 2.014651298522949
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: 0.012268661057075136
      w: 0.9999247371456846
  bin_1:
    position: 
      x: 4.319528488763594
      y: 1.976347481492174
      z: -0.004286359057586994
    orientation: 
      x: -0.0017634782644519631
      y: 0.045229535861640247
      z: 0.03898649177081485
      w: 0.99821402148515
  ball_1:
    position: 
      x: -2.271734903397791
      y: -0.824172911640978
      z: -0.004286961940725753
    orientation: 
      x: 0.0231124046454588
      y: 0.03889610667206996
      z: -0.5102127371134645
      w: 0.8588573062648105
  doorway_1:
    position: 
      x: -0.7058808574672154
      y: -0.04160877240541357
      z: -0.004291044059784005
    orientation: 
      x: -0.0011608400734263089
      y: 0.04522579422230842
      z: 0.025869815003330257
      w: 0.998641093015393
  doorway_1_blue:
    position: 
      x: 0.3962286114692688
      y: 0.09294455498456955
      z: -0.004291044059784005
    orientation: 
      x: -0.0011608400734263089
      y: 0.04522579422230842
      z: 0.025869815003330257
      w: 0.998641093015393

      
